lang/python/python3-host.mk

```diff
@@ -11,7 +11,6 @@
 # For PYTHON3_VERSION
 python3_mk_path:=$(dir $(lastword $(MAKEFILE_LIST)))
 include $(python3_mk_path)python3-version.mk
-
```

Comment:
nitpick: i like an empty line after an include; i'd leave it;
no major objection to remove it;